README.md

```diff
@@ -0,0 +1,66 @@
+# terraform-aws-backup
```

Comment:
Please have some basic description what this module does and some simple example code here, have also tests/basic folder with some basic setups example